src/main/java/org/opensearch/flowframework/transport/ProvisionWorkflowTransportAction.java

```diff
@@ -126,15 +126,11 @@ protected void doExecute(Task task, WorkflowRequest request, ActionListener<Work
 
                 flowFrameworkIndicesHandler.updateFlowFrameworkSystemIndexDoc(
                     workflowId,
-                    ImmutableMap.of(
-                        STATE_FIELD,
-                        State.PROVISIONING,
-                        PROVISIONING_PROGRESS_FIELD,
-                        ProvisioningProgress.IN_PROGRESS,
-                        PROVISION_START_TIME_FIELD,
-                        Instant.now().toEpochMilli(),
-                        RESOURCES_CREATED_FIELD,
-                        Collections.emptyList()
+                    Map.ofEntries(
+                        Map.entry(STATE_FIELD, State.PROVISIONING),
+                        Map.entry(PROVISIONING_PROGRESS_FIELD, ProvisioningProgress.IN_PROGRESS),
+                        Map.entry(PROVISION_START_TIME_FIELD, Instant.now().toEpochMilli()),
+                        Map.entry(RESOURCES_CREATED_FIELD, Collections.emptyList())
```

Comment:
Why did this list shorten?

Reply:
As well as the changes below?

Reply:
It's pairing things up.   There were 8 single entries, there are now 4 pairs.

The previous code was: `ImmutableMap.of(k1, v1, k2, v2, k3, v3, k4, v4)`.  
The new code separates the key-value pairs for readability: `Map.ofEntries(Map.entry(k1, v1), Map.entry(k2, v2), Map.entry(k3, v3), Map.entry(k4, v4))`.